### DISCUSSION TITLE
core dump

### DISCUSSION THREAD

**Discussion by michaelc95** [madMAx43v3r/chia-plotter#469] [Q&A]
Phase 2 took 272.88 sec
Wrote plot header with 268 bytes
terminate called after throwing an instance of 'std::runtime_error'
  what():  thread failed with: fopen() failed
Aborted (core dumped)

Any ideas? I am running the latest version downloaded today. I looked around the Chia-plotter folder but I dont see a core file. 

./chia_plot -n 1 -r 48 -u 256 -t /mnt/raid16/ -2 /mnt/ramdisk/ -d /mnt/raid8/  

raid16 is a 16 way raid 0 10k sas
raid 8 is an 8 way raid 0 10k sas
ramdisk is a 110gb tmpfs ramdisk
Dual Xeon 2690v3 with 24 core 48 threads and 256 gb of ram. 

Also tried bumping up my ramdisk to 150gb, no change.. still core dumped same message

**Comment by snoby** [comment]
Seeing this with the latest update I just pulled.  Worked yesterday, I tried changing to 128 and to 256 buckets with no change.

This commit is working on another machine of mine still.
16f9793b78b60eb9a4e84ff8048998c3ac62a0e5

**Reply by madMAx43v3r** [reply]
it's fixed in latest master

**Reply by JerkyChew899** [reply]
Can confirm that this is fixed, at least on my system. Today is the first time ever successfully completing a plot with MadMax.

**Reply by XIII39** [reply]
I was having this error on one of my plotters today. This particular machine has 128GB of G.Skill RAM as opposed to the Kingston or Corsair in the other plotters. I had to disable the XMP in the BIOS to get it to work

**Comment by joelrb** [comment]
Oh so this is a common issue. I got segmentation fault (core dump). Stopped at [P1] Table 4.

**Comment by ESP4Ever** [comment]
Hello,

I would recommend you to check that you have enough RAM/disk space. I got this error using a fully RAM disk plotting: the final file stays on the disk until copy/rename/delete phase complete and there is no pause between plotting cycles. My network bandwidth is not enough to transmit 109GB file quickly (it takes around 16 minutes) so I was getting the RAM overflow at the end of the first -beginning of the second phase
